src/services/Worker.rs

```diff
@@ -35,34 +37,57 @@ pub async fn post_start_worker(
   
   let request_body = StartWorker::build_query(variables);
   
-  return post_to_nvaapi::<
+  let bad_json_on_resp = post_to_nvaapi::<
     crate::start_worker::Variables,
     crate::start_worker::ResponseData,
-    String
+    crate::start_worker::ResponseData
   >(
     nvacl,
     request_body, 
-    |s| {
-      // let sm = serde_json::from_str(s).unwrap();
-      if let Some(sw) = &s.start_worker {
-        match serde_json::from_str::<serde_json::Map<String,serde_json::Value>>(sw) {
-          Ok(res) => {
-            //FIXME, make more robust -- ensure fields are present etc.
-            return res["id"].to_string();
-          },
-          Err(e) => {
-            to_console_error(&format!("start worker parse response error: {:?}", e));
-            return "".to_string();
-          }
-        }
-      }
-      return "".to_string();
-    },
+    |s| s,
     Some(1)
   ).await;
-}
 
+  match bad_json_on_resp {
+    Ok(res) => {
+      // to_console_debug(&format!("post_start_worker response: {:?}", &res));
+      return Uuid::parse_str(&res.start_worker.unwrap().to_string()).map_err(|e_| Box::new(e_) as Box<dyn Error>);
+    },
+    Err(e) => {
+      let re = Regex::new("[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}").unwrap();
```

Comment:
@Affie , issue as startWorker returns an JSON as Object (not string) like other JSON scalars.  So had to hack around it via regex.  Can consolidate and fix this later as part of more general work on:
- #37 

Lets keep the workaround in place for the time being.